src/overlays/actors/ovl_Eff_Zoraband/z_eff_zoraband.h

```diff
@@ -3,14 +3,17 @@
 
 #include "global.h"
 
+#define EFFZORABAND_GET_F(thisx) ((thisx)->params & 0xF)
+
 struct EffZoraband;
 
 typedef void (*EffZorabandActionFunc)(struct EffZoraband*, GlobalContext*);
 
 typedef struct EffZoraband {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x4];
-    /* 0x0148 */ EffZorabandActionFunc actionFunc;
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ s16 transparency;
+    /* 0x146 */ u16 transpInit;
```

Comment:
As mentioned in Discord, this seems more a set of flags of which they happened to use only one.